recipes/yt/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.2.3" %}
+
+package:
+  name: yt
+  version: {{ version }}
+
+source:
+  fn: yt-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/y/yt/yt-{{ version }}.tar.gz
+  sha256: 4d6ccf345d9fab965335c9faf8708c7eea79366b81d77f0f302808be3e82c0ed
+
+build:
+  entry_points:
+    - yt = yt.utilities.command_line:run_main
+
+requirements:
+  build:
+    - python
+    - numpy
+    - cython
+    - setuptools 
+    - libpython # [win]
+    - mingw # [win]
+  run:
+    - python
+    - numpy
+    - h5py
+    - sympy
+    - matplotlib
+    - ipython
+
+test:
+  requires:
+    - nose
+  commands:
+    - yt -h
+  imports:
+    - yt
+
+about:
+  home: http://yt-project.org/
+  license: BSD
+
+extra:
+    recipe-maintainers:
+      - Nathan Goldbaum
+      - Kacper Kowalik
```

Comment:
So, here we use GitHub handles. This has to do with how permission are assigned in conda-forge to feedstocks.